airflow/config_templates/config.yml

```
@@ -2035,6 +2035,17 @@
       type: string
       example: ~
       default: ""
+    - name: delete_option_kwargs
+      description: |
+        Optional keyword arguments to pass to the ``delete_namespaced_pod`` kubernetes client
+        ``core_v1_api`` method when using the Kubernetes Executor.
+        This should be an object and can contain any of the options listed in the ``v1DeleteOptions``
+        class defined here:
+        https://github.com/kubernetes-client/python/blob/41f11a09995efcd0142e25946adc7591431bfb2f/kubernetes/client/models/v1_delete_options.py#L19
+      version_added: ~
+      type: string
+      example: "{{\"grace_period_seconds\": 10}}"
```

Doesn't affect the output, but could be done as:

```suggestion
      example: '{{"grace_period_seconds": 10}}'
```

Also: OH GOD. Do we show this as `{` or `{{` when we render the docs. Becasuse it needs to be `{` -- `{{` is only needed in default_airflow.cfg because that is a template, but we should show in our docs is not templated/formated.

---

Maybe we need to update the pre-commit script to replace `{` with `{{` and `}` with `}}` so that this and the default_airflow.cfg are correct.

---

We need `{{"grace_period_seconds": 10}}` in airflow.cfg because ConfigParser does not parse `{` properly

---

It is tricky because of `task_log_prefix_template = {{ti.dag_id}}-{{ti.task_id}}-{{execution_date}}-{{try_number}}`. It would replcae it too `task_log_prefix_template = {{{{ti.dag_id}}}}-{{{{ti.task_id}}}}-{{{{execution_date}}}}-{{{{try_number}}}}`

> Maybe we need to update the pre-commit script to replace `{` with `{{` and `}` with `}}` so that this and the default_airflow.cfg are correct.

---

Updated !